scripts/unix-install.sh

```diff
@@ -451,8 +463,20 @@ install_package()
   stop_service
   succeeded
 
+  proxy_args=""
+  if [ -n "$proxy" ]; then
+    proxy_args="-x $proxy"
+    if [ -n "$proxy_user" ]; then
+      proxy_args="$proxy_args -U $proxy_user:$proxy_password"
+    fi
+  fi
+
+  if [ -n "$proxy" ]; then
+    info "Downloading package using proxy..."
+  fi 
+
   info "Downloading binary..."
-  curl -L "$agent_download_url" -o "$agent_binary" --progress-bar --fail || error_exit "$LINENO" "Failed to download package"
+  curl -L "${proxy_args}" "$agent_download_url" -o "$agent_binary" --progress-bar --fail || error_exit "$LINENO" "Failed to download package"
```

Comment:
Need to use `eval` here, when inserting arguments. For example, `proxy_args` expands to `-x <proxy ip> -U username:password`. The -x and -U will not behave:

without eval
```
scripts/unix-install.sh --proxy squid:3128 --proxy-user user --proxy-password password
```
```
...
===================================================
| Configuring Installation Variables
===================================================
  Using proxy from arguments: 10.33.105.14:3128
  Configuration complete!

===================================================
| Installing Stanza
===================================================
  Creating Stanza directory...
    Succeeded!
  Checking that service is not running...
    Succeeded!
  Downloading package using proxy...
  Downloading binary...

curl: (5) Could not resolve proxy:  10.33.105.14
    ERROR (scripts/unix-install.sh:479): Failed to download package
```

With eval, I get no error and my proxy logs the connection.

```suggestion
  eval curl -L "${proxy_args}" "$agent_download_url" -o "$agent_binary" --progress-bar --fail || error_exit "$LINENO" "Failed to download package"
```